driver-core/src/main/com/mongodb/operation/MixedBulkWriteOperation.java

```diff
@@ -221,19 +243,25 @@ private boolean serverSupportsWriteCommands(final ConnectionDescription connecti
     }
 
     private Iterable<Run> getRunGenerator(final ConnectionDescription connectionDescription) {
+        Boolean bypassValidation = null;
```

Comment:
What's the rationale for this?  The default is null, so as long as it's not set by the user, no harm done.  And I don't see this check in the other operations.